### DISCUSSION TITLE
winning icons in 1.6.2

### DISCUSSION THREAD

**Discussion by Duckpogs** [KadeArchive/Kade-Engine#1877] [Q&A]
How do I add winning icons in 1.6.2?

**Comment by napstaa967** [comment]
first add all the winning icons to the icon files, make then the third icon in the image, once done:

healthicon.hx:
```hx
	public function changeIcon(char:String)
	{
		if (char != 'bf-pixel' && char != 'bf-old')
			char = char.split("-")[0];

		loadGraphic(Paths.image('icons/icon-' + char), true, 150, 150);
		if(char.endsWith('-pixel') || char.startsWith('senpai') || char.startsWith('spirit'))
			antialiasing = false
		else
			antialiasing = FlxG.save.data.antialiasing;
		animation.add(char, [0, 1, 2], 0, false, isPlayer); //add a 2 after the 1 here
		animation.play(char);
	}
```

playstate.hx
```hx
if (healthBar.percent < 20)
			iconP1.animation.curAnim.curFrame = 1;
			iconP2.animation.curAnim.curFrame = 2; //add
		else
			iconP1.animation.curAnim.curFrame = 0
			iconP2.animation.curAnim.curFrame = 1; //add

		if (healthBar.percent > 80)
			iconP2.animation.curAnim.curFrame = 1;
			iconP1.animation.curAnim.curFrame = 2; //add
		else
			iconP2.animation.curAnim.curFrame = 0;
			iconP1.animation.curAnim.curFrame = 1; //add
```

**Reply by libracrumbs** [reply]
I need this on 1.6, can you tell how?

**Reply by napstaa967** [reply]
> I need this on 1.6, can you tell how?

for 1.6 iirc they still had the icons in the same file so:

for every character add a new win icon next to it, the width and height has to be 150

once ordered count from 0 starting from the first icon and modify healthicon.hx, add to the arrays where the icon is specified the number of the icons

lets say gf's icons are 0, 1, 2

then you set that as the array, you will have to move the rest foward by 1
